tutorials/rabi-oscillations.md

```diff
@@ -0,0 +1,183 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# QuTiP example: Vacuum Rabi oscillations in the Jaynes-Cummings model
+
+Authors: J.R. Johansson and P.D. Nation
+
+Slight modifications: C. Staufenbiel (2022)
+
+This notebook demonstrates how to simulate the quantum vacuum rabi 
+oscillations in the Jaynes-Cumming model, using the Master Equation Solver 
+`qutip.mesolve`. We also consider the dissipative version of the 
+Jaynes-Cumming model, i.e., the cavity and the atom are coupled to an 
+environment.
+
+
+For more information on the theory behind the Master Equation Solver see [the documentation](https://qutip.org/docs/latest/guide/dynamics/dynamics-master.html#non-unitary-evolution).
+
+
+### Package import
+
+```python
+%matplotlib inline
+import matplotlib.pyplot as plt
+import numpy as np
+from qutip import *
```

Comment:
I'd like to avoid using `from qutip import *` in the tutorial notebooks, largely to discourage others from copying this style. It is convenient, but it makes knowing where some functions come from tricky, and perhaps more worryingly, we will likely break some of this in version 5 in order to avoid having to import all of QuTiP into the top-level package (although there are other solutions to that problem, like lazy module interfaces, I'm not sure how well those interact with `import *`).